documentation/src/docs/asciidoc/release-notes/release-notes-5.5.0-M2.adoc

```diff
@@ -25,6 +25,8 @@ on GitHub.
 
 * `AnnotationSupport.findRepeatableAnnotations()` now finds repeatable annotations used as
   meta-annotations on other repeatable annotations.
+* Exception thrown by `TestExecutionListeners` no longer cause test execution to abort.
```

Comment:
Note to self: There's an `s` missing here.

Reply:
@marcphilipp Just one doubt. Should I update with `Note to self:` comments? Or these comments are for you only?

Reply:
@gopinath-langote Sorry for taking so long to reply. Those were meant for myself to fix before merging this which I'll do as soon as CI passes.